pkg/controller/clusterissuers/sync.go

```diff
@@ -18,13 +18,6 @@ const (
 
 func (c *Controller) Sync(ctx context.Context, iss *v1alpha1.ClusterIssuer) (err error) {
 	issuerCopy := iss.DeepCopy()
-	i, err := c.issuerFactory.IssuerFor(issuerCopy)
-
-	if err != nil {
-		return err
-	}
-
-	err = i.Setup(ctx)
```

Comment:
Making the previously mentioned change to validate config in Setup(), we can revert the changes here and in issuers/sync.go